packages/block-editor/src/components/block-list/block-mobile-toolbar.native.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { Keyboard, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import styles from './block-mobile-toolbar.scss';
+import BlockMover from '../block-mover';
+import InspectorControls from '../inspector-controls';
+
+const BlockMobileToolbar = ( {
+	clientId,
+	onDelete,
+} ) => (
+	<View style={ styles.toolbar }>
+		<BlockMover clientIds={ [ clientId ] } />
+
+		<View style={ styles.spacer } />
```

Comment:
I think that flexbox allows for this without the extra spacer view, although I haven't managed to do it directly on the inspector slot without a wrapper view:

```jsx
<View style={ styles.inspectorControls }>
	<InspectorControls.Slot />
</View>
```

```scss
.inspectorControls {
	flex-grow: 1;
	align-items: flex-end;
}
```

Reply:
I think it makes sense with flexbox. It's not like we're floating our elements left and right, we're pushing it to the sides thanks to a spacer.
A good alternative would be to make 2 groups of icons, each using flex as well and justifying content to the left and right. I did experiment a bit with it but I saw some strange behavior with the toolbar disappearing sometimes. The current solution seems to be working fine for us...